### PR TITLE
Bugfix: S3C-2571 headObject accept-ranges header

### PR DESCRIPTION
**Pull request by ilkescality**
HeadObject supports "Accept-ranges" header for request with `Range` (String | i.e. "bytes=0-9") param.

### CODE REVIEW

**Comment by ilkescality** [comment]
Closing this one due to hitting a corner issue, creating a new one https://github.com/scality/cloudserver/pull/2454

File: lib/utilities/collectResponseHeaders.js [hunk 1]
```diff
@@ -45,6 +45,9 @@ function collectResponseHeaders(objectMD, corsHeaders, versioningCfg,
         responseMetaHeaders['x-amz-server-side-encryption-aws-kms-key-id']
             = objectMD['x-amz-server-side-encryption-aws-kms-key-id'];
     }
+    if (objectMD['accept-ranges']) {
```
Comment (jonathan-gramain) [inline lib/utilities/collectResponseHeaders.js:48]: Why is this part of `objectMD`? For me it's a server-side capability that is not related to object internal metadata.
